### ISSUE TITLE
Use export with string template if else error.

### ISSUE THREAD

**Issue by ngocgodwar** [pigpigyyy/MoonPlus#13]
This code below throw compile error:
```moonscript
export var

isHero = -> true
message = "I am #{if isHero! then "a HERO" else "just a random dude"}!"
print message
```
Please check it out. Thanks for your time and regards!

**Comment by pigpigyyy** [comment]
The minimum example to reproduce the problem could be
```Moonscript
export x
f do 123
```
The problem is the return statement getting transformed in some inner function scopes, and those function scopes are treated as the the root function scope. The export statement forbid using return statement under the root function scope. And now the problem got fixed with commit eb86243480988263b02765f5fe097cc1811507d0

**Comment by ngocgodwar** [comment]
Confirm Fixed!
Thanks for your time and regards!
